### DISCUSSION TITLE
Auto Scan on Boot

### DISCUSSION THREAD

**Discussion by OneBangDash** [egzumer/uv-k5-firmware-custom#415] [Q&A]
Is there a way to set my device to automatically start scanning when I power it on? My previous device had this feature and I find myself forgetting to long hold scan when I turn it on.

**Comment by AubsUK** [comment]
I'm currently working on a change for scanning lists (adding 10 lists not just the two), and this was one option I was thinking of adding  at the same time if I can get it working.

**Reply by OneBangDash** [reply]
This would be amazing! I'd love to have additional scan lists as well.

**Reply by AubsUK** [reply]
https://github.com/egzumer/uv-k5-firmware-custom/discussions/480

**Reply by AubsUK** [reply]
@OneBangDash - I've just finished the firmware and CHIRP driver over on my repo - https://github.com/AubsUK/uv-k5-firmware-custom

Hope you like it.  Open a discussion there if you have any suggestions.

**Reply by OneBangDash** [reply]
I've been using this for a few weeks now and it's been working great! I'm not using all of the new scan lists but having auto-scan when I turn on my device is awesome. Thank you!
